likecoin/js/admin-settings/src/components/SubmitButton.js

```diff
@@ -0,0 +1,13 @@
+import { __ } from '@wordpress/i18n';
+
+function SubmitButton() {
+  return (
+    <tr style={{ textAlign: 'left' }}>
+      <button className="likecoinSubmitButton">
+        {__('Confirm', 'likecoin')}
+      </button>
+    </tr>
```

Comment:
Why wrapping `<button/>` with `<tr/>`?

Reply:
so button can be in the left while inner text is align-center

Reply:
changed it from <tr /> to <div />